test/test_kubeclient.rb

```diff
@@ -34,18 +34,12 @@ def test_no_path_in_uri
     assert_equal('http://localhost:8080/api/v1', rest_client.url.to_s)
   end
 
-  def test_no_version_passed
```

Comment:
How about we leave this test in and assert it raises that ArgumentError?

Reply:
done

Reply:
Rubocop complaining now 👮

Reply:
fixed!